src/GafferImage/Warp.cpp

```diff
@@ -602,6 +609,14 @@ void Warp::hashChannelData( const GafferImage::ImagePlug *parent, const Gaffer::
 IECore::ConstFloatVectorDataPtr Warp::computeChannelData( const std::string &channelName, const Imath::V2i &tileOrigin, const Gaffer::Context *context, const ImagePlug *parent ) const
 {
 	ConstCompoundObjectPtr sampleRegions = sampleRegionsPlug()->getValue();
+
+	{
+		Gaffer::ContextPtr sampleRegionsContext = new Gaffer::Context( *context, Gaffer::Context::Borrowed );
+		sampleRegionsContext->remove( ImagePlug::channelNameContextName );
+		Gaffer::Context::Scope sampleRegionsScope( sampleRegionsContext.get() );
+		sampleRegions = sampleRegionsPlug()->getValue();
```

Comment:
Gah, it's late, that last comment was gibberish.  The reason why this stupid mistake didn't affect runtime too much was that the hash was still computed correctly, so the bogus getValue call would still hit the cache ( the time for this plug was actually compute dominated ).  Anyway, fixed now.